tests/README.md

```diff
@@ -13,11 +13,11 @@ Note: Not all tests have been converted to tox and/or pytest yet.
 
 ## Running the tests
 
-Before running vdsm tests, activate the environment:
+Before running Vdsm tests, activate the environment (which you need to create once, with `make venv`, as described in [doc/development.md](doc/development.md)):
```

Comment:
Please keep lines wrapped. The text should look nice in your editor since
developer will check this file locally.

Reply:
Ok. What is our recommended line length?

Reply:
In general 79 chars, but for long text using shorter lines may be better.
for example in commit message I use 72 chars.